python/_modules/grpc.html

```diff
@@ -6,7 +6,7 @@
   <head>
     <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    <title>grpc &#8212; gRPC Python 1.18.0 documentation</title>
+    <title>grpc &#8212; gRPC Python 1.19.0 documentation</title>
```

Comment:
I've been burned before, so I check this line everytime. Haha.

Reply:
I learnt that too!